ui/pages/settings/flask/snaps-list-tab/index.scss

```diff
@@ -0,0 +1,29 @@
+.snap-list-tab {
+  width: 100%;
+  height: 100%;
+
+  &__wrapper {
+    width: auto;
+  }
+
+  &__body {
+    padding: 12px 18px;
+
+    @media screen and (min-width: $break-large) {
+      padding: 12px;
+    }
+  }
+
+  .snap-settings-card {
+    margin: 8px 0;
+    max-width: 344px;
+  }
+
+  &__container--no-snaps {
+    display: flex;
+    width: 100%;
+    height: 100%;
+    justify-content: center;
+    align-items: center;
+  }
+}
```

Comment:
[REQ] A good chunk of these styles can be achieved using Box and its props. Could we attempt to eliminate some of the extra CSS?